abs.yaml

```
@@ -0,0 +1,2 @@
+channels:
+  - https://staging.continuum.io/prefect/fs/jpeg-feedstock/pr7/3f0a23b
```

```suggestion
  - https://staging.continuum.io/prefect/fs/jpeg-feedstock/pr7/0c4591b
```
3f0a23b is gone, so I am not sure which version of jpeg it was.

---

updated jpeg is uploaded now so I deleted abs.yaml